src/network/message.rs

```diff
@@ -62,15 +58,11 @@ impl CommandString {
 }
 
 impl fmt::Display for CommandString {
-    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        f.write_str(self.0.as_ref())
-    }
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result { f.write_str(self.0.as_ref()) }
```

Comment:
Highlighting this because it might be too long for folks now? This is caused by `fn_single_line = true` combined with `max_width = 100`.